docs/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  min-height: 100vh;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 2rem;
+  background: #282c34;
```

Comment:
Are we planning to link these styles to the existing theme at all? Or give them their own theme/structure?

Reply:
Nah. GitHub Pages won't compile our docs. They need to be just straight-up client-side code.